PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetQnVectors.h

```diff
@@ -75,12 +74,15 @@ class AliAnalysisTaskJetQnVectors : public AliAnalysisTaskSE {
     bool fEnableTPCPhiVsCentrDistr;                  /// flag to enable histos of phi vs. centr
     bool fEnableQvecTPCVsCentrDistr;                 /// flag to enable histos of TPC Q-vector vs. centr
 
-    AliJEQnVectorHandler* fJEQnVecHandler;           /// Qn-vector handler
+    AliJEQnVectorHandler* fJEQnVecHandler1;           /// Qn-vector handler
+    AliJEQnVectorHandler* fJEQnVecHandler2;           /// Qn-vector handler
     int fHarmonic;                                   /// Qn-vector harmonic
     int fCalibType;                                  /// type of calibrations used by handler
     int fNormMethod;                                 /// normalisation of Q vector
 
     TString fOADBFileName;                           /// OADB input file name
```

Comment:
Based on the changes in the constructor, I think you meant to remove this variable since it's replaced by the other two? You should also update the variable set in `SetUseAODBCalibrations1(...)` as appropriate

Reply:
Thanks for catching that typo with the SetUseAODBCalibrations - I've just fixed that. I'm not sure I'm looking in the right spot for the other comment. Which variable are you referring to?

Reply:
You replaced `fOADBFileName` in the constructor with `fOADBFileName1` and `fOADBFileName2` (ie. `fOADBFileName` is not even initialized anymore). It's only settable via `SetUseAODBCalibrations(...)`, but neither `fOADBFileName1` or `fOADBFileName2` are settable. So I assumed that you wanted to replace `fOADBFileName` with those two everywhere. Right now, you seem to have kind of a half and half split. If you want to keep `fOADBFileName`, you should include initialize it in the constructors. If you don't, then I would remove it entirely, and make `fOADBFileName1` and `fOADBFileName2` settable.